server/routes/journey/intercept/unknownPreRecallSentenceTypeController.ts

```diff
@@ -46,14 +46,22 @@ export default class UnknownPreRecallSentenceTypeController implements Controlle
       recallId,
       RecallJourneyUrls.manualJourneyStart.name,
     )
-    const continueLink = `${config.urls.remandAndSentencing}/person/${nomsId}`
+
+    const sentenceUuids = Array.from(
```

Comment:
You can just use `isPossible.sentenceIds` here.

Reply:
thanks - good spot!